README.md

```diff
@@ -73,8 +73,9 @@ CLI usage:
 
 ```
 $ ./unstringify
-Usage: ./unstringify inputPath outputPath
+Usage: ./unstringify inputPath outputPath templatePath
```

Comment:
I would put templatePath in brackets, to make it clear it is optional:
`Usage: ./unstringify inputPath outputPath [templatePath]`

Reply:
Done!